docs/learn-accounts.md

```diff
@@ -95,7 +95,23 @@ Alternatively, you might find some wallets on the
 [Polkaproject.com page](http://www.polkaproject.com/) but bear in mind that these are **unaudited**
 and not officially affiliated with Web3 Foundation or the Polkadot project unless otherwise stated.
 
-Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
+Full hardware wallet integration on [Ledger](https://ledger.com) is coming soon.
+
+### Compatibility
+
+Seed phrases are not portable across all wallets. Some wallets only support certain signing curves
+and derivation paths. See the following table for compatibility.
+
+|                  | sr25519 |  ed25519  | secp256k1 |
+| :--------------- | :-----: | :-------: | :-------: |
+| Polkadot JS UI   |    X    |     X     |     X     |
+| Polkadot JS Apps |    X    |     X     |     X     |
+| Parity Signer    |    X    |    X\*    |           |
+| Ledger           |         | X&dagger; |           |
+| Subkey           |    X    |     X     |     X     |
+
+- \* Planned
+- &dagger; Only with BIP32 derivations
```

Comment:
Maybe we need some other form more explicitly telling which kind of deriations (hard or soft and using which protocol) each of the implementations uses?
So far this table doesn't answers the question "OK, I have seed and a derivation path from Ledger, which tools I can use to properly import it"?
It seems that Polkadot JS UI would support the BIP32-derivation, but that (as far as I can tell) is not correct.

Reply:
Does Ledger have documentation about these paths? In this case, I think the Wiki should point people to the resources needed to figure out what is portable. There's no way we can keep up with every wallet's implementation. I would rather use this to warn people that portability is an issue and they need to look at X and Y resources to figure out if they can port a seed phrase from A to B.

Reply:
Makes sense. @jleni, do you have some Kusama/Polkadot-on-Ledger documentation we can point to from here?